### PR TITLE
Calendar view

### PR DESCRIPTION
**Pull request by malissaromero**

### CODE REVIEW

**Comment by AliTopal89** [comment]
Calendar page with basic crud functionality

- User can create a lesson with a title, body text that is short, and a start time and end time
 
- The `start_time` and `end_time` format currently allows you to enter values in this format `10AM on DD/MM/YYYY` so for example `10AM on 23/05/2017`

- The date time format validates the presence of `start_time` and `end_time`

- The user can update the the page but put the date time format the same way the user created it 

- When the user goes back to the index page they can see the event title and when clicked can see what the lesson is about

File: app/models/calendar_item.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class CalendarItem < ApplicationRecord
+  validates_presence_of :start_time, :end_time
```
Comment (osondoar) [inline app/models/calendar_item.rb]: `end_time` can be blank.

File: app/views/admin/calendar_items/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  <dd><%= @calendar_item.title %></dd>
+
+  <li>Body</li>
+  <dd><%= @calendar_item.body %></dd>
+
+  <li>Start Date</li>
+  <dd><%= @calendar_item.start_time %></dd>
+
+  <li>End Date</li>
+  <dd><%= @calendar_item.end_time %></dd>
```
Comment (osondoar) [inline app/views/admin/calendar_items/show.html.erb]: I _think_ end date is not needed, we can hide it for now.

File: app/models/calendar_item.rb (hunk 1 above)
Comment (osondoar) [reply app/models/calendar_item.rb]: and `Title` is a required field

**Comment by osondoar** [comment]
added a couple of comments about validations. Apart from that 👍 . Good job!

**Comment by iamanissa** [comment]
@osondoar suggestions make sense. I concur. Other than that, looks good.

**Pull request merged**